web/js/codeworld.js

```diff
@@ -160,7 +160,22 @@ function initCodeworld() {
             "Ctrl-Up": changeFontSize(1),
             "Ctrl-Down": changeFontSize(-1)
         },
-        textHover: onHover
+        textHover: onHover,
+        gutters: ["CodeMirror-lint-markers"],
+        lint: {
+            getAnnotations: function(text, callback, cm){
```

Comment:
cm seems to be unused here.  In JavaScript you can drop unused parameters from the end of a list.

Reply:
Thanks, removed.